concepts/packages/about.md

```diff
@@ -1,12 +1,37 @@
 # About
 
-In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package is preferred to be the same as the folder it is located in.
+In Go an application is organized in packages.
+A package is a collection of source files located in the same folder.
+All source files in a folder must have the same package name at the top of the file.
+The package is preferred to be the same as the folder it is located in.
 
 ```go
 package greeting
 ```
 
-Standard library packages are imported using their name. Other packages are imported using the url of the package's location. A package can be given an alias when importing. This can be used to avoid package name conflicts:
+Go provides an extensive standard library of packages which you can use in your program using the `import` keyword.
+Standard library packages are imported using their name.
+
+```go
+package greeting
+
+import "fmt"
+```
+
+Multiple packages can be imported at once by encoding them in brackets.
```

Comment:
```suggestion
Multiple packages can be imported at once by encoding them in parentheses.
```

I believe `[` is the (square) bracket, `(` is a parenthesis, `{` is a brace or curly bracket, and `<` is an inequality sign or pointy bracket. These are parentheses.